components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/password/NotificationPasswordRecoveryManager.java

```diff
@@ -1289,35 +1309,13 @@ private boolean isAskPasswordEmailTemplateTypeExists(String tenantDomain) {
         return templateType != null;
     }
 
-    /**
-     * Retrieve email address of the user.
-     *
-     * @param user      User the email need to be retrieved.
-     * @return email address of the user.
-     * @throws IdentityRecoveryServerException
-     */
-    private String getEmail(User user) throws IdentityRecoveryServerException {
```

Comment:
getEmail function is moved to the Util class as getUserClaim function to use with both email and mobile claims.